dallinger/command_line/docker.py

```diff
@@ -208,9 +208,9 @@ def deploy_image(image, mode, config_options):
     dashboard_password = secrets.token_urlsafe(8)
     dallinger_uid = str(uuid.uuid4())
     config_dict = {
-        "aws_access_key_id": config.get("aws_access_key_id"),
-        "aws_secret_access_key": config.get("aws_secret_access_key"),
-        "aws_region": config.get("aws_region"),
+        "AWS_ACCESS_KEY_ID": config.get("aws_access_key_id"),
+        "AWS_SECRET_ACCESS_KEY": config.get("aws_secret_access_key"),
+        "AWS_REGION": config.get("aws_region"),
```

Comment:
I believe it should be `AWS_DEFAULT_REGION`. [See boto docs](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/configuration.html#using-environment-variables).